verify-signatures.markdown

```diff
@@ -28,6 +29,12 @@ Our current PGP key fingerprint is `C5B7 4989 65EF D1C2 924B  A9D5 39D3 1987 931
 
 *Note: If you see "gpg: Note: This key has expired!" as originally noted in [Issue 2040](https://github.com/opensearch-project/opensearch-build/issues/2040){:target="_blank"}, please download the newest key. See change log for dates.*
 
+### <a name="JarSigner"></a> JarSigner
```

Comment:
```suggestion
### <a name="JarSigner">JarSigner</a>
```
Are links not created automatically for headers?

Reply:
No it didn't work without it. https://github.com/opensearch-project/project-website/pull/1730/commits/7cc73864ea73fb7aa9662e395dc50afa101a2751